lib/typeUtils.ts

```diff
@@ -20,3 +20,5 @@ export type InferRequestFromContract<
   Querystring: InferredOptionalSchema<Contract['requestQuerySchema']>
   Reply: InferredOptionalSchema<Contract['successResponseBodySchema']>
 }>
+
+// I am not sure if I understand the purpose of this, isn't it what we have with fastify api contracts?
```

Comment:
it is, but if someone wants to move specific handler to a separate, they could use this to type their request.

that said, they could use `buildFastifyRouteHandler` helper function in that case, so yeah, most likely this is redundant. I will drop this for now and see if we miss it